frontend/src/component/executiveDashboard/FlagsProjectChart/FlagsProjectChart.tsx

```diff
@@ -0,0 +1,5 @@
+import { lazy } from 'react';
+
+export const FlagsProjectChart = lazy(
```

Comment:
do we do this for each chart? maybe we can lazy load all charts that show up together